kubernetes/iceberg/kind.yaml

```diff
@@ -0,0 +1,29 @@
+# This configuration creates a 5-node kind cluster optimized for the streaming lab:
+#
+# 1. Control Plane Node:
+#    - Runs Kubernetes API server, etcd, scheduler, and controller manager
+#    - Port mapping (32090) exposes MinIO console for external access
+#    - Isolated from workload execution for stability
+#
+# 2. Worker Nodes (4x):
+#    - Distribute Redpanda brokers, MinIO storage, Spark executors, and Jupyter
+#    - Multiple nodes allow testing of distributed scenarios and node failures
+#    - Sufficient capacity for concurrent Spark jobs and streaming workloads
+#
+# 3. Port Mapping Strategy:
```

Comment:
Just my two cents, I really not a fan of using port mapping with KiND for things like that. `kubect port-forward` is more generally applicable (e.g. you could use the same instructions against a GKE cluster and it'll work the same way), it doesn't rely on magic numbers, nor will it failure if there are port conflicts on the host machine.